extensions-contrib/opentelemetry-extensions/src/main/java/org/apache/druid/data/input/opentelemetry/protobuf/OpenTelemetryMetricsProtobufReader.java

```diff
@@ -221,8 +228,10 @@ InputRow createRow(long timeUnixMilli, Map<String, Object> event)
   }
 
   @Override
-  public CloseableIterator<InputRowListPlusRawValues> sample()
+  public CloseableIterator<InputRowListPlusRawValues> sample() throws IOException
```

Comment:
same here we can avoid throwing IOException here now

Reply:
replied above. It's required by `try-with-resources`